example/index.html

```diff
@@ -0,0 +1,171 @@
+<!doctype html>
+<html>
+  <head>
+    <title>Halogen Datepicker Example</title>
+    <style>
+
+body {
+  font-family: sans-serif;
+  margin: auto;
+}
+
+.ColorPicker {
```

Comment:
Well, I don't know about others but when I start using a lib I usually just drop it to project and try to make it works, styles, layouts and other customizations are the last things I worry about during startup. That's why I think having nice defaults with predefined styles is useful.